objects/ogamemanager/Step_0.gml

```diff
@@ -63,7 +63,9 @@ if (global.gameTimer == 0)
 		{
 			global.p2Rounds++;
 		}
-		var particle = instance_create_layer(0, 0, "KO_Text", oParticles);
+		
+		// Display Time's up text
+		var particle = instance_create_layer(global.camObj.x-80, 0, "KO_Text", oParticles);

```

Comment:
```global.camObj``` should probably be renamed to something more descriptive like ```global.mainCamera``` or similar, but I won't hold back an approval because this has 21 references in 12 different files, so changing it and testing it would take more time than I think is warranted. But something to keep in mind.